pkgs/development/tools/ocaml/camlp4/default.nix

```diff
@@ -43,6 +43,10 @@ stdenv.mkDerivation rec {
   dontAddPrefix = true;
 
   preConfigure = ''
+    # increase stack space for spacetime variant of the compiler
+    # https://github.com/ocaml/ocaml/issues/7435
+    ulimit -s unlimited
```

Comment:
Does this work on all platforms?

Could this change be made only when “spacetime” support is enabled? (To this end, it might be useful to inherit `spaceTimeSupport` in the `passthru` attribute of the `ocaml` derivation.)

Reply:
I expect so, because this is a bash builtin.

Reply:
CI just failed on darwin:

> /nix/store/mmv9acmxa9g4p9fk3ck2kh4hii7ym48f-stdenv-darwin/setup: line 91: ulimit: stack size: cannot modify limit: Operation not permitted